build.gradle.kts

```diff
@@ -17,8 +17,9 @@ dependencies {
     implementation("junit:junit:4.13.1")
     implementation("org.junit.jupiter:junit-jupiter:5.8.1")
     implementation("org.testng:testng:7.1.0")
-    testImplementation(kotlin("test"))
+    implementation("com.github.javafaker:javafaker:1.0.2")
```

Comment:
André pra coseguir adicionar essa mesma lib só que na mão você precisa seguir esse guia :point_down: 
https://www.geeksforgeeks.org/how-to-add-external-jar-file-to-an-intellij-idea-project/
Lembrando que você tem que ter baixado o `.jar` da lib :eyes: